scripts/docker/common.sh

```diff
@@ -95,6 +95,18 @@ function common::get_constraints_location() {
         python_version="$(python --version 2>/dev/stdout | cut -d " " -f 2 | cut -d "." -f 1-2)"
         AIRFLOW_CONSTRAINTS_LOCATION="${constraints_base}/${AIRFLOW_CONSTRAINTS_MODE}-${python_version}.txt"
     fi
+
+    if [[ ${AIRFLOW_CONSTRAINTS_LOCATION} =~ http.* ]]; then
+        echo
+        echo "${COLOR_BLUE}Downloading constraints from ${AIRFLOW_CONSTRAINTS_LOCATION} to ${HOME}/constraints.txt ${COLOR_RESET}"
+        echo
+        curl -o "${HOME}/constraints.txt" "${AIRFLOW_CONSTRAINTS_LOCATION}" >/dev/null
```

Comment:
small nit

```suggestion
        curl -s -o "${HOME}/constraints.txt" "${AIRFLOW_CONSTRAINTS_LOCATION}"
```

Reply:
Or silent + show error

```suggestion
        curl -Ss -o "${HOME}/constraints.txt" "${AIRFLOW_CONSTRAINTS_LOCATION}"
```

Reply:
yep. Actually `-sf` is best.

Reply:
`-sSf` ???

-S will show an error, e.g.

```console
❯ curl -sSf https2://raw.githubusercontent.com/apache/airflow/constraints-main/constraints-source-providers-3.9.txt
curl: (1) Protocol "https2" not supported or disabled in libcurl
❯ curl -sSf https://raw.githubusercontent.com/apache/airflow/constraints-main/constraints-source-providers-3.13.txt
curl: (22) The requested URL returned error: 404
```

```console
❯ curl -sf https://raw.githubusercontent.com/apache/airflow/constraints-main/constraints-source-providers-3.13.txt
❯ curl -sf https2://raw.githubusercontent.com/apache/airflow/constraints-main/constraints-source-providers-3.9.txt
```

Reply:
Interesting : 

```
root@8d95a3223a2c:/opt/airflow# curl --help
Usage: curl [options...] <url>
 -d, --data <data>          HTTP POST data
 -f, --fail                 Fail fast with no output on HTTP errors
 -h, --help <category>      Get help for commands
 -i, --include              Include protocol response headers in the output
 -o, --output <file>        Write to file instead of stdout
 -O, --remote-name          Write output to a file named as the remote file
 -s, --silent               Silent mode
 -T, --upload-file <file>   Transfer local FILE to destination
 -u, --user <user:password> Server user and password
 -A, --user-agent <name>    Send User-Agent <name> to server
 -v, --verbose              Make the operation more talkative
 -V, --version              Show version number and quit

This is not the full help, this menu is stripped into categories.
Use "--help category" to get an overview of all categories.
For all options use the manual or "--help all".
```

Reply:
Byt yeah. It works like you wrote :)

Reply:
`curl --help all` shows it :)

Reply:
I've always thought that curl has a over 9000 different keys 🤣